experiments/EclipseExample/checkForDep.sh

```diff
@@ -0,0 +1,28 @@
+mvn compile
```

Comment:
What does this script do? When and how should we invoke this?

Reply:
Please remove the file or comment out the stuff that shouldn't be ran to make the script useful.